app/Models/Process.php

```diff
@@ -10,6 +10,8 @@ class Process extends Model
     use SoftDeletes;
 
     const MAX_WITNESSES = 3;
+    const BRAZILIAN_MINIMUM_WAGE = 998.00;
```

Comment:
```suggestion
    const BRAZI_MINIMUM_WAGE = 998.00;
```

Mudar nos demais lugares